docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+```scala
+import com.databricks.labs.overwatch.pipeline.Migration
+val sourceETLDB = "ow_bronze_migration_etl"
+val migrateRootPath = "/mnt/overwatch_playground/721-snap/migration"
+val configPath = "abfss://overwatch-field-playground@overwatchglobalinternal.dfs.core.windows.net/sourav/configs/721-migration"
+val tablesToExclude = "notebook_silver:notebook_gold"
+Migration.process(sourceETLDB,migrateRootPath,configPath)
+```
+We need below parameters to run the Migration process through Databricks Notebook:
+
+| Param           | Type     | Optional   | Description                                                                                                                           |
+|-----------------|----------|------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| sourceETLDB     | String   | No         | Source Database Name for which Migration need to be done.                                                                             |
+| migrateRootPath | String   | No         | Target path to where migration need to be performed.                                                                                  |
+| configPath      | String   | No         | Configuration Path where the config file for the source is present. Path can be CSV file or delta path ot delta table.                |
+| tablesToExclude | String   | Yes        | Array of table names to exclude from the snapshot. This is the table name only - without the database prefix. By Default it is empty. |
+
+After Migration process is done the tables from sourcePrefix will be restored in the `migrateRootPath/globalshare` location
+![Migration](/images/DataEngineer/Migration_path.png)
+
+
+##### Migration through Databricks Job
+Below is the Configuration for the Databricks Job
+1. **Type** - Jar
+2. **Main class** - com.databricks.labs.overwatch.pipeline.MigrationRunner
+3. **Parameters for the Job** Same as mentioned above for `Migration through Databricks Notebook`
+
+
+![Migration_Job](/images/DataEngineer/Migration_Job.png)
+
+#### Validation Functionality
+This is a special module in Migration Process. By default, it is part of Migration.process() but if user want to run it separately they can run it separately
+to check whether the configuration they would provide for Migration process, are correct or not.
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Migration
+val sourceETLDB = "ow_bronze_migration_etl"
+val configPath = "abfss://overwatch-field-playground@overwatchglobalinternal.dfs.core.windows.net/sourav/configs/721-migration"
+Migration.isValid(sourceETLDB,configPath)
+```
+Upon running the above function you will get proper output whether the input parameters you would be using for Migration process,passed the validation check or not. One example of the same is as below:
```

Comment:
Change to: "This function will return true if the configuration is valid, and will also output relevant info for the fields checked. For example:"

Reply:
Done